tools/go.mod

```diff
@@ -3,15 +3,41 @@ module github.com/buildpacks/lifecycle/tools
 go 1.14
 
 require (
```

Comment:
Why are the tools `go.mod` and `go.sum` modified? It doesn't seem like windows base image change should require any dependency additions/upgrades?

Reply:
@ekcasey  That's a good question...I'm not sure either.  When I ran `make test`, these files were updated.  Should I undo these changes?